src/index.html

```diff
@@ -10,7 +10,44 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+    <form
+      action="#"
+      class="form"
+      method="post"
+      data-qa="big"
+    >
+      <input
+        type="search"
+        data-qa="keypress"
+        class="search-bar"
+        placeholder="Try &quot;Los Angeles&quot;"
+        name="search__bar"
```

Comment:
6 point in checklist

Reply:
What means six points?

Reply:
you need to fix this
```suggestion
        name="searchBar"
```
you should assign values to name attribute in camelCase